link/src/main/java/com/stripe/android/link/ui/paymentmethod/FormViewModel.kt

```diff
@@ -54,14 +53,6 @@ internal class FormViewModel @Inject internal constructor(
         }
     }
 
-    private val sectionToFieldIdentifierMap = formSpec.items
```

Comment:
@brnunes-stripe Can you verify that you are not hiding things by an IdentifierSpec representing a section.   You can still hide by a particular field name

Reply:
Yes, this is only used by the card form, so only the postal code field is hidden.